src/content/docs/accounts/accounts-billing/general-account-settings/intro-account-settings.mdx

```diff
@@ -11,7 +11,7 @@ redirects:
 - /docs/accounts/original-accounts-billing
 ---
 
-To find your account management and user management settings, go to [one.newrelic.com](https://one.newrelic.com) and, in the upper right corner, click the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown).
+To find your account management and user management settings, go to [one.newrelic.com](https://one.newrelic.com) and click the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown).
```

Comment:
This is a short doc, so I'm wondering other ways you can flesh out this intro paragraph. I'm wondering if you can give a short sentence summary that overviews common problems users would go to these docs for. An easy way could be coming up with a user story and then kneading that story into usable content. An example:

"Want certainty about monthly billing cycles? Our [doc name] goes into XYZ" where XYZ are common questions people might want to know.

This change can be a follow up change after you push this ^ edit through. Just a suggestion!